client/components/tinymce/plugins/media/plugin.jsx

```diff
@@ -632,16 +632,20 @@ function mediaButton( editor ) {
 
 		gallery = assign( {}, MediaConstants.GalleryDefaultAttrs, gallery.attrs.named );
 
-		gallery.items = gallery.ids.split( ',' ).map( ( id ) => {
-			id = parseInt( id, 10 );
+		gallery.items = [];
 
-			const media = MediaStore.get( selectedSite.ID, id );
-			if ( ! media ) {
-				MediaActions.fetch( selectedSite.ID, id );
-			}
+		if ( gallery.ids ) {
+			gallery.items = gallery.ids.split( ',' ).map( ( id ) => {
+				id = parseInt( id, 10 );
```

Comment:
It's outside the scope of this pull request, but I imagine funny things will happen if IDs aren't the sort of values we expect them to be (non-numeric). Maybe more pertinent to this pull request is the case that the `ids` attribute is set to an empty string.

I might imagine we could resolve both issues with some simple validation:

```js
gallery.items = ( gallery.ids || '' )
	.split( ',' )
	.reduce( ( memo, id ) => {
		id = parseInt( id, 10 );
		
		if ( id > 0 ) {
			const media = MediaStore.get( selectedSite.ID, id );
			if ( ! media ) {
				MediaActions.fetch( selectedSite.ID, id );
			}

			memo.push( assign( { ID: id }, media ) )
		}

		return memo;
	}, [] );
```